pennylane/devices/default_qubit.py

```diff
@@ -634,11 +673,11 @@ def execute_and_compute_derivatives(
                         _adjoint_jac_wrapper,
                         vanilla_circuits,
                         seeds,
-                        [self._prng_key] * len(vanilla_circuits),
+                        self.get_prng_keys(num=len(vanilla_circuits)),
```

Comment:
Do we even need to be sending seeds when calculating derivatives? adjoint is analytic only, so we aren't going to be doing any sampling 🤔

Reply:
Good point. `rng`s, `seed`s removed.